File: LeetCode-Programs/A Number After a Double Reversal.cpp
class Solution {
public:
    bool isSameAfterReversals(int num) {
        int temp=num;
        int rev=0;
        while(temp>0)
        {
            rev=rev*10 + temp%10;
            temp=temp/10;
        }
        temp=rev;
        int temp2=0;
        while(temp>0)
        {
            temp2=temp2*10+temp%10;
            temp=temp/10;
        }
        if(num==temp2)
            return true;
        else
            return false;
    }
};

File: LeetCode-Programs/A Number After a Double Reversal.py
#TLE

class Solution:
    def isSameAfterReversals(self, num: int) -> bool:
        temp=num
        rev=0
        while temp>0:
            rev=rev*10 + temp%10
            temp=temp/10
        temp=rev
        temp2=0
        while temp>0:
            temp2=temp2*10 + temp%10
            temp=temp/10
        if num==temp2:
            return True
        else:
            return False
        
        
        
        
        
        
        
 class Solution:
    def isSameAfterReversals(self, num: int) -> bool:
        '''temp=num
        rev=0
        while temp>0:
            rev=rev*10 + temp%10
            temp=temp/10
        temp=rev
        temp2=0
        while temp>0:
            temp2=temp2*10 + temp%10
            temp=temp/10
        if num==temp2:
            return True
        else:
            return False'''
        
        '''rev=int(str(num(::-1)))
        reve=int(str(rev(::-1)))
        if num==reve:
            return True
        else:
            return False'''
        return num==int(str(int(str(num)[::-1]))[::-1])
